core/group/service.go

```diff
@@ -187,7 +192,11 @@ func (s Service) ListAdmins(ctx context.Context, idOrSlug string) ([]user.User,
 func (s Service) AddAdmins(ctx context.Context, groupIdOrSlug string, userIds []string) ([]user.User, error) {
 	currentUser, err := s.userService.FetchCurrentUser(ctx)
 	if err != nil {
-		return []user.User{}, err
+		return []user.User{}, fmt.Errorf("%w: %s", user.ErrInvalidEmail, err.Error())
+	}
+
+	if len(userIds) < 1 {
```

Comment:
this `if` block can be merged to the above error handling block (to keep all error handling together)